detection-rules/spam_fake_photo_share.yml

```diff
@@ -17,7 +17,14 @@ source: |
       strings.ilike(subject.subject, "*picture*", "*photo*", "*image*")
       or strings.icontains(subject.subject, sender.display_name)
       or (
-        strings.ilike(body.html.display_text, "*picture*", "*photo*", "*image*")
+        (
```

Comment:
I haven't developed an eye for this yet, but are the double open/close parentheses required/intentional?

Reply:
Nice 👀